src/js/angular/import/directives/import-resource-tree.directive.js

```diff
@@ -190,28 +189,29 @@ function importResourceTreeDirective($timeout, ImportContextService) {
              * and can have their states reset.
              */
             const setCanResetResourcesFlag = () => {
-                $scope.canResetSelectedResources = $scope.selectedResources
-                    .some((treeResource) => treeResource.importResource.status === ImportResourceStatus.DONE);
+                $scope.canResetSelectedResources = $scope.resources.getAllSelected()
+                    .some((treeResource) => {
```

Comment:
This is better also to be moved in the resources model, e.g. `resources.getSelectedImportedResources()`

Reply:
Done